### ISSUE TITLE
Statistiques - pourcentage de personnes qui cliquent sur "Voir toutes les action""

### ISSUE THREAD

**Issue by Clem-mrtn** [datagir/nosgestesclimat-site#688]
Connaître le nombre / pourcentage de personnes qui ont fini le test et qui cliquent sur le bouton *"Voir toutes les actions"*

**Comment by laem** [comment]
Aaah d'ac :) Je rouvre alors.

**Comment by Clemog** [comment]
Yes mais à pas oublier de prioriser pour le pipe de la semaine du coup, c'est pas négligeable la gestion de la page stats en temps je trouve
